test/test.gyp

```diff
@@ -162,6 +162,9 @@
         'test-udp-multicast-interface6.c',
         'test-udp-try-send.c',
         'test-uname.c',
+        'test-not-writable-after-shutdown.c',
```

Comment:
sort alphabetically

Reply:
Is is worthwhile to sort the whole list or to make a separate PR for doing that?